modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/resolver/ResolverUtils.java

```diff
@@ -121,7 +122,14 @@ private ResolverUtils() {
    * @return the given {@code value} but converting a {@link CursorProvider} to a {@link Cursor} if any is present.
    */
   public static Object resolveCursor(Object value) {
-    return resolveCursor(value, identity());
+    // For performance reasons, we avoid delegating to the other version with the valueMapper parameter.
```

Comment:
Actually with this implementation I can remove the other versions that receive the `valueMapper` parameter, because they are no longer used. Do you think I should do it? or does it make sense to keep them in this utilitary class?
(My vote would be in favor of the removal)

Reply:
The unused functions were removed